distrepos/__main__.py

```diff
@@ -205,6 +206,24 @@ def update_repo_timestamp(options: Options):
         f.write(datetime.now().strftime("%a %d %b %Y %H:%M:%S %Z"))
 
 
+def link_static(options: Options) -> int:
+    """
+    Create symlinks from the main data directory to the static data directory
+    """
+
+    _log.info("Updating symlinks to static-data directory")
+    try:
+        ok, err = link_static_data(options)
+        if ok:
+            _log.info("static-data symlinks updated successfully")
+            return 0
+        else:
+            _log.warning(f"Unable to update static-data symlinks: {err}")
+            return ERR_FAILURES
+    except Exception as e:
+        _log.error(f"Unexpected error updating static-data symlinks: {e}")
```

Comment:
```suggestion
        _log.exception(f"Unexpected error updating static-data symlinks: {e}")
```
if we didn't expect the exception, we should probably get a stack trace.